pkg/handler/asset.go

```diff
@@ -195,6 +195,7 @@ func (u *UserSelectHandler) proxyAsset(asset model.PermAsset) {
 		Account:       selectedAccount.Alias,
 		Protocol:      protocol,
 		ConnectMethod: "ssh",
+		InputUsername: selectedAccount.Username,
 		RemoteAddr:    u.h.sess.RemoteAddr(),
 	}
 	if selectedAccount.IsInputUser() {
```

Comment:
There is no clear indication of any specific code changes, function name mismatches, potential issues, or optimizations you're referring to. It's unclear what part of the codebase this question pertains to. If there were differences in functionality that needed review, they would be highlighted clearly in your message.

However, if you specify which functions or areas within the same module need clarification on whether things work as expected, I can provide more personalized advice.